docs/source/index.rst

```diff
@@ -9,15 +9,10 @@ Welcome to the MAAP User Documentation!
 .. toctree::
   :maxdepth: 1
   
-  intro.rst
-  searching.rst
-  visualizing.rst 
-  accessing.rst
-  querying.rst
-  userdata.rst
+  getting_started.rst
+  science_examples.rst
   tutorials.rst
-  faqs.rst
```

Comment:
FAQs is delisted here, but the FAQ folder still exists. Is this intentional to hide pages (orphan)? More generally should the folder and section names match?

Reply:
I moved the FAQ into the System Reference Guide section. Maybe I forgot to move the `faq/` folder along with `faqs.rst`

Reply:
yup that's it. so maybe just a move is in order, since these are clearly FAQs only for System section.

Reply:
Oh I think I left it at the top level because all the image-links have to be altered for the faq files to work. I'll move it and update them.

(This is why I was asking about the `_static` folder and whether I can make new ones to move entire sub-trees of files around.)

Reply:
fixed in commit: 4854358c3f32ec484b1dc25ae356b734e88b6d0f

Reply:
I also have a subsequent commit to rename faqs.rst to faq.rst for consistency